### DISCUSSION TITLE
Parameters of PolynomialStaticLoad

### DISCUSSION THREAD

**Discussion by mario-akira** [upb-lea/gym-electric-motor#250] [Q&A]
Hello, I would like to know if it is possible to update the load parameters using "PolynomialStaticLoad" during the control simulation. I want to update the parameters because I am simulating the dynamics of a go-kart with the parameters a, b, and c of the "PolynomialStaticLoad," so I would like to change them while simulating the PMSM motor. Is it possible to do this, or would I have to perform multiple simulations, always changing them initially?

**Comment by max-schenke** [comment]
My suggestion on that would be a little hacky but might work:

I would define a wrapper function around the env.step() in which the parameters a, b, c of the PolynomialLoad are accessed and exchanged.
This can then be performed according to time or according to measurement variables.

A standardized way of doing this is not implemented yet.

**Reply by mario-akira** [reply]
Thank you, I will try to do something like that.
